packages/ngtools/webpack/src/ngcc_processor.ts

```diff
@@ -61,18 +107,9 @@ export class NgccProcessor {
     }
 
     const packageJsonPath = this.tryResolvePackage(moduleName, resolvedFileName);
-    if (!packageJsonPath) {
-      // add it to processed so the second time round we skip this.
-      this._processedModules.add(resolvedFileName);
-
-      return;
-    }
-
```

Comment:
why remove this block?

Reply:
It’s been moved https://github.com/angular/angular-cli/pull/17105/files#diff-456f73b84b7b60a60000b590b3301f7bR112

Reply:
Oh! Then I think the comment before the line where it has been moved to needs to change. Since that only mentions readonly, I was thrown and didn't realise it also handled missing package.json files.